pkgs/development/python-modules/scales/default.nix

```diff
@@ -16,11 +17,23 @@ buildPythonPackage rec {
     sha256 = "8b6930f7d4bf115192290b44c757af5e254e3fcfcb75ff9a51f5c96a404e2753";
   };
 
-  nativeCheckInputs = [ nose ];
+  patches = [
+    # Use html module in Python 3 and cgi module in Python 2
+    (fetchpatch2 {
+      url = "https://github.com/Cue/scales/commit/ee69d45f1a7f928f7b241702e9be06007444115e.patch?full_index=1";
+      hash = "sha256-xBlgkh1mf+3J7GtNI0zGb7Sum8UYbTpUmM12sxK/fSU=";
+    })
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
   propagatedBuildInputs = [ six ];
 
-  # No tests included
-  doCheck = false;
+  preCheck = ''
```

Comment:
This should be done in postPatch so that we notice early if it doesn't apply anymore